packages/slate-commons/src/lib/decorateExtensions.ts

```diff
@@ -2,28 +2,27 @@ import { NodeEntry, Range } from 'slate';
 
 import { Decorate, Extension } from '../types';
 
-const decorateExtensions = (extensions: Extension[], decorateList: Decorate[]) => (
-    entry: NodeEntry,
-) => {
-    let ranges: Range[] = [];
+const decorateExtensions =
```

Comment:
I really wish you have not had upgraded prettier in this PR :disappointed:

Reply:
Can you please revert [this commit](https://github.com/prezly/slate/pull/28/commits/52dfca7d6c7964aa0935eda8eb9dbd8ab6aef6ab) and undo the prettier version bump? 

For the open source sake!

Reply:
e5e98d9